plots/laws/gravity/free_fall_acceleration.py

```diff
@@ -0,0 +1,29 @@
+from sympy import solve, pretty, symbols
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import (
+    units, convert_to
+)
+from symplyphysics.laws.gravity import free_fall_acceleration_from_height as acceleration
+print("Formula is:\n{}".format(acceleration.print()))
+gravity_constant=convert_to(acceleration.gravity_constant, units.newton * units.meter**2 / units.kilogram**2).subs(units.newton * units.meter**2 / units.kilogram**2, 1).evalf(5)
+solved = solve(acceleration.law, acceleration.acceleration_free_fall, dict=True)[0][acceleration.acceleration_free_fall]
+result_acceleration = solved.subs({acceleration.gravity_constant: gravity_constant,
+                                   acceleration.earth_mass: 5.976e+24,
```

Comment:
I prefer not to align parameters. It makes code lines too long. Use one level indentation instead.